.github/workflows/go.yml

```diff
@@ -16,14 +16,18 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v5
         with:
-          go-version: '1.24'
+          go-version: "1.24"
           cache: true
 
-      - name: Lint
+      # Use the golangci-lint action for improved caching etc.
+      - name: Lint with golangi-lint
```

Comment:
Why not just one step that does `make lint`? That way if we add future linters to the Makefile they will be automatically picked up.

Reply:
Yeah I kept it this way because I _think_ the golangci-lint action gives us free caching but 🤷 I've switched to `make lint` instead.